tools/rpkg/tests/testthat/_snaps/explain.md

```diff
@@ -1,71 +0,0 @@
-# EXPLAIN gives reasonable output
```

Comment:
It seems that this file required for snapshot tests was accidentally(?!) deleted. If there is no particular reason to delete, please don't.

Reply:
Yeah, I'm not sure how these got deleted, thanks for spotting it.